.github/workflows/workflow.yml

```diff
@@ -9,7 +9,7 @@ on:
   pull_request:
 
 env:
-  RELAY_CHAIN_VERSION: v0.9.16
+  RELAY_CHAIN_VERSION: v0.9.17-rc4
```

Comment:
why not just 0.9.17?

Reply:
there is no release on tag v0.9.17, this is official release https://github.com/paritytech/polkadot/releases/tag/v0.9.17-rc4